client/js/components/statement/publicStatementModal/StatementModal.vue

```diff
@@ -74,11 +74,12 @@
           <i
             aria-hidden="true"
             :class="prefixClass('c-statement__hint-icon fa fa-lg fa-info-circle')" />
-          <span
-            :class="prefixClass('block u-ml')">
+          <div
```

Comment:
Would it be feasible to use DpInlineNotification here?

Reply:
Not sure, because we have several messages here, and DpInlineNotification can only handle one. We might want to adjust DpInlineNotification correspondingly in the future, but I don't know if we want to do it now?

Reply:
Thanks for pointing it out, done in 65c257dcaff72bb9d3477db979fb449e7875e72c